### ISSUE TITLE
People on same network still can't vote even with permissive turned on

### ISSUE THREAD

**Issue by jdnguyen** [strawpoll/strawpoll#33]
I am using the API and even if I turn on permissive, some people in the same room can vote but others can't. I was wondering if I could have a mode that allows people to vote multiple time. I trust the people voting to only vote once and we might be sharing the same IP.

**Comment by dandirks** [comment]
Are you trying to vote through the API? If so, that would be why permissive mode is not working. That particular setting requires the use of the website itself.

I will look at implementing an option that turns off vote duplication checking altogether in a future release. Thanks for the suggestion.

**Comment by jdnguyen** [comment]
I am creating the poll through the API but the voting is done through the website. When the poll is created, it tells me permissive mode is set to true but when my friends in the same room try to vote, it tells them they have already voted.

**Comment by lindseyanne-zz** [comment]
im creating through the website, but i am getting vote duplication check enforced both ways. initially i thought perhaps the setting had been inverted, but i have created the same poll several times both with and without the option checked, and get the duplicate error both ways:

> Could not cast your vote. Error returned by server: User has already voted on poll

**Comment by zolk** [comment]
Same problem here. Creating a poll through the website. Permissive option is enabled, but votes from different computers on the same network are still getting rejected.

**Comment by Tonza** [comment]
Same issue here (with permissive option), but after few tries it works though.

Edit: using website directly to poll

**Comment by mitterdoo** [comment]
i'm also having this problem with the website. the poll was created with the website and voters are using the website. i AM using mobile for both, however. this needs to be fixed soon.

**Comment by dandirks** [comment]
I just verified that this is working correctly for me in both mobile and through desktop browsers. I'm not sure how to replicate what is being seen in this thread.

Do you all have cookies enabled? Is there anything that would be changing what is sent by your browser as far as your IP/cookies/headers?

**Comment by mitterdoo** [comment]
yes i have cookies enabled. to narrow it down, im using an iPhone 5C if that helps. i am ticking the permissive voting option upon creation and the first vote is sent correctly and shows up on the results, but i can't vote anymore after that

**Comment by dandirks** [comment]
Are you trying to vote again from the same device? It is expected that this
is disallowed.

**Comment by jdnguyen** [comment]
It's not working for me across a whole network. I would create a poll and send it to everyone in my office. Most people would tell me that they are unable to vote.

**Comment by dandirks** [comment]
Hmm... I'm not sure how to get that to happen.

Well, this weekend I will attempt to implement a new option to just
entirely disable duplication checking. That should serve as a decent
workaround for those that are unable to get the permissive checking to work.

**Comment by dandirks** [comment]
Just an update: I have completed development for the feature to allow disabling duplication checking. I want to do a bit more testing to make sure that this is working correctly and that I did not break any other functionality, but if that all pans out, this feature should hopefully be live sometime early this week.

**Comment by dandirks** [comment]
The ability to disable the duplication checking has been pushed live. Apologies for the delay.

**Comment by divinity76** [comment]
its a flawed check, this is a big problem in schools/universities, where thousands of students share 1 global ip (shared across many local ips & wifis), only the first student will get to vote, the rest will be blocked! perhaps add support for authenticating through facebook, google+, or even github?
